src/hooks/v2/V2ContractLoader.ts

```diff
@@ -9,6 +9,23 @@ import { useContext, useEffect, useState } from 'react'
 import { readProvider } from 'constants/readProvider'
 import { readNetwork } from 'constants/networks'
 
+/**
+ *  Defines the ABI filename to use for a given V2ContractName.
+ */
+const CONTRACT_ABI_OVERRIDES: { [k in V2ContractName]?: string } = {
+  JBController: 'JBController_2',
```

Comment:
Here, we say which ABI file we want to use for a given contract name.

The new contracts have the `_2` suffix